src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exceptions.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final int ID_COLUMN = 1;
+    private static final int NAME_COLUMN = 2;
+    private static final int COUNTRY_COLUMN = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String sqlRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(sqlRequest)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                manufacturer.setId(generatedKeys.getObject(ID_COLUMN, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert "
+                    + manufacturer + " data to the database.", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String sqlRequest = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(sqlRequest)) {
+            ResultSet resultSet = statement.executeQuery();
+            return Optional.of(setManufacturer(resultSet));
```

Comment:
don't throw exceptions when optional must be empty, and set necessary parameters to query
```suggestion
        try (Connection connection = ConnectionUtil.getConnection();
                PreparedStatement statement = connection.prepareStatement(sqlRequest)) {
            ResultSet resultSet = statement.executeQuery();
            if (resultSet.next()) {
            return Optional.of(setManufacturer(resultSet));
            }
```